### PR TITLE
Add twia14

### PR DESCRIPTION
**Pull request by torkleyy**
There are some TODOs, maybe @jojolepro you can fix them if you have enough time for that?

### CODE REVIEW

File: src/content/blog/twia-14.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+author_link = "https://github.com/torkleyy"
++++
+
+Hello and welcome to the 14th issue of _This Week in Amethyst_, a blog bringing to
+you the latest changes regarding Amethyst, a data-driven game engine written in
+Rust.
+
+Content of this issue:
+
+* recent changes
```
Comment (LucioFranco) [inline src/content/blog/twia-14.md]: Capitalize these maybe?

File: src/content/blog/twia-14.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+
+to a pipeline stage and you get a skybox.
+You can tweak it using the `SkyboxColor` resource.
+
+[pho]: https://github.com/photex
+
+### Font loading twaks
+
+Thank you [jjedelsky][jje] and [DoumanAsh][dou] for making font loading better.
+We can now
```
Comment (LucioFranco) [inline src/content/blog/twia-14.md:106]: This feels awkward, I would put this on a new line and use `:` at the end.

File: src/content/blog/twia-14.md [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+to [Leorii][leo] for the amazing PR! If you want to, you can already check
+out [the PR][prb] and add some comments in case you notice anything.
+
+[leo]: https://github.com/Leorii
+[prb]: https://github.com/amethyst/amethyst/pull/1085
+
+## Interesting discussions
+
+Although not exactly an Amethyst discussion, there's a thread on
+internals.rust-lang.org that discusses a working group for Rust game
```
Comment (LucioFranco) [inline src/content/blog/twia-14.md]: linkify 😄

File: src/content/blog/twia-14.md [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+to work on, don't hesitate! We are going to help you and point you into the
+right direction if something doesn't work out at first.
+
+We have tasks with different difficulty. Make sure to choose easier ones at
+first, because hard issues can become quite frustrating, especially if you
+haven't worked on Amethyst before.
+
+Here are seven issues for you to pick from:
+
+TODO
```
Comment (LucioFranco) [inline src/content/blog/twia-14.md]: Here are some from `laminar`:

https://github.com/amethyst/laminar/issues/33

https://github.com/amethyst/laminar/issues/45

**Review by LucioFranco — COMMENTED** [review]
Looking good! 👍

File: src/content/blog/twia-14.md [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+```rust
+.with_pass(DrawSkybox::new())
+```
+
+to a pipeline stage and you get a skybox.
+You can tweak it using the `SkyboxColor` resource.
+
+[pho]: https://github.com/photex
+
+### Font loading twaks
```
Comment (re-l124c41plus) [inline src/content/blog/twia-14.md]:
```suggestion
### Font loading tweaks
```
Found a small typo

Comment (torkleyy) [reply src/content/blog/twia-14.md]: Nice catch, thanks!

**Review by LucioFranco — APPROVED** [review]
lookin good!

**Comment by torkleyy** [comment]
Skybox screenshot (pasting this here to get a permanent link):

image

**Review by Moxinilian — APPROVED** [review]

**Comment by Moxinilian** [comment]
You could also add the sreenshot to the static folder, maybe in `static/assets/blog/twia14`, which would then host it at `amethyst.rs/assets/blog/twia14/skybox.png`.

**Review by Rhuagh — APPROVED** [review]

**Comment by LucioFranco** [comment]
@torkleyy were you gonna merge this today?

**Comment by torkleyy** [comment]
image

**Comment by torkleyy** [comment]
Yes

**Pull request merged**